packages/backend/src/routes/info.mjs

```diff
@@ -1,32 +1,15 @@
-import {
-  UNIREP_ADDRESS,
-  ETH_PROVIDER_URL,
-  DB_PATH,
-  VERSION,
-  RELEASE,
-  EMPTY_EPOCH_TREE_ROOT,
-  AGGREGATE_KEY_COUNT,
-  STATE_TREE_DEPTH,
-  EPOCH_TREE_DEPTH,
-  EPOCH_TREE_ARITY,
-  EPOCH_KEY_NONCE_COUNT,
-} from '../config.mjs'
+import { UNIREP_ADDRESS, ETH_PROVIDER_URL } from '../config.mjs'
 import catchError from '../helpers/catchError.mjs'
 
 export default ({ app, db, synchronizer }) => {
   const handler = async (req, res) => {
     res.json({
       UNIREP_ADDRESS,
       ETH_PROVIDER_URL,
-      DB_PATH,
-      VERSION,
-      RELEASE,
-      EMPTY_EPOCH_TREE_ROOT,
-      AGGREGATE_KEY_COUNT,
```

Comment:
I see that you're getting the circuit config values from the synchronizer, but where will we get these from?

Reply:
The synchronizer loads the values from the smart contract

Reply:
Oh, `EMPTY_EPOCH_TREE_ROOT`, `AGGREGATE_KEY_COUNT` are removed in the latest version. As for version/release I think we can exclude these for now, or hardcode them in the frontend.